File: Common/Entity/src/main/java/it/ey/dto/AutoritaApprovatoreDTO.java
package it.ey.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.SuperBuilder;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@SuperBuilder(toBuilder = true)
public class AutoritaApprovatoreDTO extends BaseDTO
{
    private Long id;
    private String codice;
    private String testo;
}

File: dfp-piao-fe/src/app/shared/models/classes/autorita-approvatore-dto.ts
import { CampiTecniciDTO } from './campi-tecnici-dto';

export class AutoritaApprovatoreDTO extends CampiTecniciDTO {
  id!: number;
  codice!: string;
  testo!: string;
  idPiao?: number;
  statoSezione?: string;
}
